wordpress-mobile/WordPress-Android
# Quick Start - Guided Navigation - Fix Snackbar Delays

Parent #15126
Fixes "Delayed Snackbars" issue mentioned in https://github.com/wordpress-mobile/WordPress-Android/issues/15258.

**To test**

- Go to My Site tab
- Select a site with quick start in progress
- Select a quick start task and complete it so that quick start notice with "Go" action button is shown
- Click "Go" action in the shown snackbar
- Notice that the next task snackbar appears immediately along with the quick start dot

**Merge Instructions**

1. Make sure #15256 is merged
2. Remove `Not Ready for Merge` label
3. Merge the PR

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
